csrc/host_ir/host_ir.cpp

```diff
@@ -21,7 +21,7 @@ namespace hir {
 
 HostUnit::HostUnit(IrBuilderPasskey passkey, std::unique_ptr<Fusion> fusion)
     : Expr(passkey), fusion_(std::make_unique<Fusion>(*fusion)) {
-  NVF_ERROR(passkey.ir_container_->isA<hir::HostIrContainer>()); // NOLINT
+  NVF_ERROR(passkey.ir_container_->isA<HostIrContainer>()); // NOLINT
```

Comment:
ditto as I mentioned in https://github.com/NVIDIA/Fuser/pull/2970#discussion_r1783846247

Reply:
I created #3087, which I think is a better way to suppress the warning.

Reply:
thanks, I applied it to `Syncrhonize`'s constructor